### ISSUE TITLE
libcloudproviders

### ISSUE THREAD

**Issue by returntrip** [flathub/org.nextcloud.Nextcloud#49]
@juliushaertl is it technically possible to integrate [libcloudproviders](https://blog.juliushaertl.de/index.php/2017/08/28/gsoc-final-report/) for better shell integration? I guess it is not possible cause of flatpak sandboxing but worth asking the experts :).

I can use Topicons Plus to display the icon but that is not really how NC should integrate in Gnome 3. Plus I need to `flatpak override --nosocket=wayland org.nextcloud.Nextcloud` to have the icon display on Fedora Silverblue.

**Comment by juliushaertl** [comment]
Yes, unfortunately flatpak is currently the blocking part here, but there has been some progress in https://gitlab.gnome.org/World/libcloudproviders/merge_requests/40

**Comment by returntrip** [comment]
Thanks for the update! I will check what is blocking this over @ Gnome Gitlab.

**Comment by geez0x1** [comment]
Starting from here https://github.com/nextcloud/desktop/issues/1289 I ended up here. While most people manage to get a tray icon (under Shell) using Topicons Plus, I can't (Debian Testing). I've also tried the Ubuntu appindicators Shell extension, which gets me the Steam icon, but to no avail. It starts successfully, but doesn't show the main window and I can't access it either. So my only way of configuration is the config files :frowning_face:
